info/available-rules.md

```diff
@@ -1,131 +1,131 @@
+| 2    | 2.1.1    | KDOC_NO_CONSTRUCTOR_PROPERTY_WITH_COMMENT | Check: warns if there is a comment before the property in the constructor                                                                                                                                                                                              | yes | no                                                                                                                                                                                                                         |                                                                                                                                                                                                                               |
+| 2    | 2.1.1    | COMMENTED_BY_KDOC                         | Check: warns if there is a kdoc comment in the code block                                                                                                                                                                                                          | yes | no                                                                                                                                                                                                                         | replace "/**" to "/*"                                                                                                                                                                                                         |
+| 2    | 2.1.2    | KDOC_WITHOUT_PARAM_TAG                    | Check: warns if an accessible method has parameters and they are not documented in KDoc<br>Fix: If accessible method has no KDoc, the KDoc template is added                                                                                                        | yes | no                                                                                                                                                                                                                         | Should also make a separate fix, even if there is already some Kdoc in place                                                                                                                                                    |
```

Comment:
"Should also make a separate fix, even if there is already some Kdoc in place" What does this sentence mean? Explanation is needed.

Reply:
Unclear to me either, let's leave as-is.